jbmc/src/java_bytecode/README.md

```diff
@@ -41,9 +41,32 @@ To be documented.
 
 To be documented.
 
-\subsection java-bytecode-genereic-substitution Generic substitution
-
-To be documented.
+\subsection java-bytecode-generic-substitution Generic substitution
+
+Generic types are resolved at runtime in Java. Therefore, when we initialize
```

Comment:
That's not quite true. At runtime there are no generic types at all, only raw types. Generic types are only used at compile time by the type checker. 
However, throwing away all this type information would be very inconvenient for our analysis. Therefore, we substitute the raw types.

Reply:
Thanks, I reformulated this accordingly, let me know what you think.